package.json

```diff
@@ -14,7 +14,7 @@
     "lint:fix": "next lint --fix",
     "prettier": "prettier -w \"{src,cypress,mocks,scripts}/**/*.{ts,tsx,css,js}\"",
     "fix": "yarn lint:fix && ts-prune && yarn prettier",
-    "test": "DEBUG_PRINT_LIMIT=30000 jest",
+    "test": "cross-env TZ=CET DEBUG_PRINT_LIMIT=30000 jest",
```

Comment:
why do we need this change?

Reply:
This sets the timezone of Jest.

Reply:
and the `cross-env` allows it to be injected in Windows machines 👍